server/controllers/apply/people/offencesController.test.ts

```diff
@@ -45,6 +47,8 @@ describe('OffencesController', () => {
 
       expect(response.render).toHaveBeenCalledWith('applications/people/selectOffence', {
         pageHeading: `Select index offence for ${person.name}`,
+        errors: {},
+        errorSummary: [],
```

Comment:
are we actually describing the unhappy path as well as the happy path?

I was expecting to see a scenario along the lines of 

```ts
expect(response.render).toHaveBeenCalledWith('applications/people/selectOffence', {
        ...
        errors: errorsAndUserInput.errors,
        errorSummary: errorsAndUserInput.errorSummary,
        ...errorsAndUserInput.userInput,
      })
```

Reply:
@edavey added